openssl/sha_test.go

```diff
@@ -81,3 +122,32 @@ func BenchmarkHash8Bytes(b *testing.B) {
 		h.Sum(sum[:0])
 	}
 }
+
+func BenchmarkSHA256(b *testing.B) {
+	b.StopTimer()
+	buf := make([]byte, 8192)
+	size := 1024
+	b.StartTimer()
+	b.SetBytes(int64(size))
+	b.ReportAllocs()
+	for i := 0; i < b.N; i++ {
+		SHA256(buf[:size])
```

Comment:
Why is `buf` longer than `size` for this benchmark?

Reply:
Hmm, I copy-pasted some code here and forgot to change the size.